paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc

```diff
@@ -11,13 +11,14 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
-
 #include <mkldnn/include/mkldnn_types.h>
+#include <stdlib.h>
```

Comment:
Why are you adding here this header with no other changes in this file? Is it really needed?